### PR TITLE
Update formula forms dev guide with new attributes

### PR DESCRIPTION
**Pull request by mateiw**
Document changes introduced by https://github.com/uyuni-project/uyuni/pull/1527.

### CODE REVIEW

File: modules/salt/pages/formulas-custom.adoc [hunk 1]
```diff
@@ -440,7 +442,79 @@ select_something:
     - option2
 ----
 
+$visible::
+Allows you to show a field or group if a condition is met. The condition must be written using the jexl[https://github.com/TomFrost/jexl] expression language.
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
Allows you to show a field or group if a condition is met. 
You must use the jexl[https://github.com/TomFrost/jexl] expression language to write the condition.
```

File: modules/salt/pages/formulas-custom.adoc [hunk 2]
```diff
@@ -440,7 +442,79 @@ select_something:
...
+An example condition is:
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
For example:
```

File: modules/salt/pages/formulas-custom.adoc [hunk 3]
```diff
@@ -440,7 +442,79 @@ select_something:
...
++
+----
+formValues.some_group.anouther_group.my_checkbox == true
+----
++
+Absolute paths always begin with ``formValues``.
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
Absolute paths must begin with ``formValues``.
```

File: modules/salt/pages/formulas-custom.adoc [hunk 4]
```diff
@@ -440,7 +442,79 @@ select_something:
...
++
+The field with this attribute associated with it will be shown only when the condition is met.
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
The field with this attribute will only show if the condition is met.
```

File: modules/salt/pages/formulas-custom.adoc [hunk 5]
```diff
@@ -440,7 +442,79 @@ select_something:
...
+In this example the field will be shown only if ``my_checkbox`` is checked.
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
In this example, the field will show only if ``my_checkbox`` is checked.
```

File: modules/salt/pages/formulas-custom.adoc [hunk 6]
```diff
@@ -440,7 +442,79 @@ select_something:
...
+The ability to use conditional statements is not limited to check boxes.
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
You can also choose other elements for the conditional statement, such as select fields or text fields.
```

File: modules/salt/pages/formulas-custom.adoc [hunk 7]
```diff
@@ -440,7 +442,79 @@ select_something:
...
+It may also be used to check values of select-fields, text-fields, and similar.
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
```

File: modules/salt/pages/formulas-custom.adoc [hunk 8]
```diff
@@ -440,7 +442,79 @@ select_something:
...
++
+A check box should be structured like this:
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
Use this structure for a check box conditional:
```

File: modules/salt/pages/formulas-custom.adoc [hunk 9]
```diff
@@ -440,7 +442,79 @@ select_something:
...
+  $type: group
+
+  another_group:
+    $type: group
+
+      my_checkbox:
+        $type: boolean
+----
++
+Relative paths can be specified using ``this.parent.value``. This references to the value of the parent.
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
Specify relative paths using ``this.parent.value`` to define the value of the parent.
```

File: modules/salt/pages/formulas-custom.adoc [hunk 10]
```diff
@@ -440,7 +442,79 @@ select_something:
...
+To refer to the parent of the parent one can use ``this.parent.parent.value``.
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
You can also refer to the parent of the parent, with ``this.parent.parent.value``.
```

File: modules/salt/pages/formulas-custom.adoc [hunk 11]
```diff
@@ -440,7 +442,79 @@ select_something:
...
+
+  yet_another_group:
+    $type: group
+
+    my_text2:
+      $visible: this.parent.parent.value.another_group.my_checkbox
 
+----
++
+If you use multiple groups with the attribute, you can allow a user to select an option and show a completely different form, dependent upon the selected value.
```
Comment (Loquacity) [inline modules/salt/pages/formulas-custom.adoc]:
```suggestion
If you use multiple groups with the attribute, users can select an option that will show a completely different form, depending on the selected value.
```

File: modules/salt/pages/formulas-custom.adoc (hunk 6 above)
Comment (Loquacity) [reply modules/salt/pages/formulas-custom.adoc]: I think you missed this one :wink:

**Comment by mateiw** [comment]
@Loquacity I addressed your comments. Is there any other change needed ?

**Comment by Loquacity** [comment]
> @Loquacity I addressed your comments. Is there any other change needed ?

I think you missed one ...

**Comment by mateiw** [comment]
@Loquacity I addressed the remaining comment.

**Comment by Loquacity** [comment]
This will need to be applied to the uyuni-docs repo now.

**Pull request merged**